### PR TITLE
Add blog post for 4.3 RC 3

### PR DESCRIPTION
**Pull request by akien-mga**
Socials info:
- TL;DR:
  * We fixed all known major regressions, so we're ready for a (final?) round of testing before the stable release.
- URL: https://godotengine.org/article/release-candidate-godot-4-3-rc-3/
- Showcased project: Pixelorama, by Orama Interactive
  * itch.io: https://orama-interactive.itch.io/pixelorama
  * Steam: https://store.steampowered.com/app/2779170/Pixelorama/
  * GitHub: https://github.com/Orama-Interactive/Pixelorama/
  * Twitter: https://x.com/OramaInteractiv
  * 1.0 version released a few days ago, together with a new Steam page where the (normally free) app can be purchased to support devs
  * Picture:
image

### CODE REVIEW

File: collections/_article/release-candidate-godot-4-3-rc-3.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+## Downloads
+
+{% include articles/download_card.html version="4.3" release="rc3" article=page %}
+
+**Standard build** includes support for GDScript and GDExtension.
+
+**.NET build** (marked as `mono`) includes support for C#, as well as GDScript and GDExtension.
+- See also [C# platform support](https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/index.html#c-platform-support).
+
+If you want to test the new Windows ARM64 builds, you will now find them listed in the usual download page.
```
Comment (akien-mga) [inline collections/_article/release-candidate-godot-4-3-rc-3.md:74]: This bit depends on merging https://github.com/godotengine/godot-website/pull/885 first.
